src/valhalla_export_edges.cc

```diff
@@ -223,15 +232,18 @@ int main(int argc, char *argv[]) {
          edge.e->trans_down() || edge.e->IsTransitLine()) //these 2 should never happen
         continue;
 
-      //get the opposing edge as well
-      edge_t opposing_edge = opposing(reader, tile, edge);
-      edge_set.set(tile_set.find(opposing_edge.i.Tile_Base())->second + opposing_edge.i.id());
-      ++set;
-
       //shortcuts arent real and maybe we dont want ferries
       if(edge.e->is_shortcut() || (!ferries && edge.e->use() == Use::kFerry))
         continue;
 
+      //get the opposing edge as well (ensure a valid edge is returned)
```

Comment:
why is this moved below the `continue`? doing so makes it so that the `edge_set.set` and `++set` dont happen, and so for shortcuts and ferries we will have to look at and mark them in additional loop interations